BangazonAPI/Controllers/ProductController.cs

```diff
@@ -0,0 +1,259 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using BangazonAPI.Models;
+using Microsoft.AspNetCore.Http;
+
+namespace BangazonAPI.Controllers
+{
```

Comment:
In general go ahead and get in the habit of leaving a comment up top that states the file's author and general purpose (see: Bangazon  definition of done). Not a big deal at all, go ahead and merge, but just for future reference/ post MVP refactoring. 👍